smk/QC_2.smk

```diff
@@ -681,10 +681,8 @@ rule motus_calc_motu:
     shell:
         """
         time (
```

Comment:
Why are these two commands not in separate rules or in a rule with motus merge, are they so quick to run?

Reply:
Yes, it takes 1-2 seconds only, per command - so 3-4 seconds for the rule. I even made it a localrule because of that.